lib/openstudio/extension/runner_config.rb

```diff
@@ -3,13 +3,18 @@
 # See also https://openstudio.net/license
 # *******************************************************************************
 
+require 'bundler'
+require 'fileutils'
 require 'json'
+require 'parallel'
 
 module OpenStudio
   module Extension
     class RunnerConfig
       FILENAME = 'runner.conf'.freeze
 
+      DEFAULT_NPROC = Parallel.processor_count - 1
```

Comment:
```suggestion
      DEFAULT_NPROC = Parallel.processor_count
```

If you feel like reverting to using all threads